src/pages/[locale]/feedback.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <section class="default-layout__info default-layout__info--info-page">
+    <h1>{{ title }}</h1>
+
+    <!-- eslint-disable-next-line vue/no-v-html -->
+    <div v-html="body" />
+  </section>
+</template>
+
+<script setup lang="ts">
+import { useI18n } from "vue-i18n";
+import metaHead from "~/lib/meta-head";
+import { useStore } from "~/stores/store";
+import { useMapStore } from "~/stores/map";
+import { usePageContent } from "~/stores/pageContent";
+
+const { locale } = useI18n();
+const store = useStore();
+const pageContent = usePageContent();
+const mapStore = useMapStore();
+
+const title = computed(() => pageContent.feedbackPage[locale.value].title);
```

Comment:
easier to read if `feedbackPage` (maybe with locale) is destructured from `usePageContent()`?:

```ts
const { feedbackPage } = usePageContent();
```

And maybe also normalise to just `page` instead of `feedbackPage`?

Reply:
pages / CMS content are now treated similar to localized resources, so this became irrelevant